recipes/cigetcert/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "cigetcert" %}
+{% set version = "1.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: db7c3226552d23516aa16a767dfce8031362640fc7a94d87a0a6e90d5c8afb36
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```

These are no longer needed but applied as default
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv "
```